zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/ZbColumnFamilies.java

```diff
@@ -201,33 +201,33 @@ public enum ZbColumnFamilies implements EnumValue {
   RETRIABLE_DISTRIBUTION(98),
   DISTRIBUTION_CONTINUATION(99),
 
-  ROLES(100),
-  ENTITY_BY_ROLE(101),
-  ROLE_BY_NAME(102),
+  RESOURCES(100),
+  RESOURCE_VERSION(101),
+  RESOURCE_BY_ID_AND_VERSION(102),
+  RESOURCE_KEY_BY_RESOURCE_ID_AND_VERSION_TAG(103),
+  RESOURCE_KEY_BY_RESOURCE_ID_AND_DEPLOYMENT_KEY(104),
 
-  TENANTS(103),
-  ENTITY_BY_TENANT(104),
-  @Deprecated
-  TENANT_BY_ID(105),
+  ROLES(105),
+  ENTITY_BY_ROLE(106),
+  ROLE_BY_NAME(107),
 
-  USER_TASK_INTERMEDIATE_STATES(106),
+  TENANTS(108),
+  ENTITY_BY_TENANT(109),
+  @Deprecated
+  TENANT_BY_ID(110),
```

Comment:
❓ Why is this column family already deprecated if it wasn't released in previous versions? Shouldn't it just be deleted?

@jfriedenstab This may be out of the scope of your changes, I'll ask the relevant team on slack.

Reply:
@jfriedenstab I've asked the team, and [they agree](https://camunda.slack.com/archives/C06UYJMMETZ/p1742291910187759?thread_ts=1742291268.107619&cid=C06UYJMMETZ) that the deprecated item can be removed, and the subsequent values decremented. Would be nice if you could take this along in the PR 🙇

Reply:
Sure, I remove it :).